pkg/tfbridge/provider_test.go

```diff
@@ -733,9 +733,10 @@ func TestCheck(t *testing.T) {
 			tf:     shimv2.NewProvider(testTFProviderV2),
 			config: shimv2.NewSchemaMap(testTFProviderV2.Schema),
 		}
-		computeStringDefault := func(opts ComputeDefaultOptions1) (interface{}, error) {
+		computeStringDefault := func(opts ComputeDefaultOptions) (interface{}, error) {
+			require.Equal(t, resource.PropertyPath{"stringPropertyValue"}, opts.PropertyPath)
 			if v, ok := opts.PriorState["stringPropertyValue"]; ok {
-				return v.StringValue(), nil
+				return v.StringValue() + "!", nil
```

Comment:
This used to be a bad test, turns out this code never ran at all until `__defaults` was added in `olds`. Instead it proactively copied the past default value. Slightly odd but learning something new - it conservatively assumes a property with DefaultInfo should copy the default value if it's not found in __defaults. With the update it's obvious that the code here must run in the test.